### PR TITLE
[Startup Tuning]LPS-98464

### PR DESCRIPTION
**Pull request by tinatian**
Hi Shuyang,

According to my local test, this will save us about 1.5s.

Thanks.
Tina.

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test:search

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationIndexingExtender.java [hunk 1]
```diff
@@ -45,6 +45,10 @@
 
 	@Activate
 	protected void activate(BundleContext bundleContext) {
+		if (!_clusterMasterExecutor.isMaster()) {
```
Comment (shuyangzhou) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationIndexingExtender.java]: @tinatian I don't think this is right. The master role may change at runtime, and you won't get a chance to reopen the tracker, right?

Comment (tinatian) [reply modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationIndexingExtender.java]: @shuyangzhou Ah! You are right! I will send you a new pull for this.

**Comment by tinatian** [comment]
Resend at https://github.com/shuyangzhou/liferay-portal/pull/8033

**Pull request closed without merging**